### ISSUE TITLE
[css-contain] Remove "at-risk" for style containment

### ISSUE THREAD

**Issue by chrishtr** [w3c/csswg-drafts#6272]
Right now, the level 2 draft spec says this:

```
The following features are at-risk, and may be dropped during the CR period:

style containment and the contain: style value
```

I propose that we drop this text. The reason it exists:
* @emilio was skeptical of the use cases for style containment
* Concerns about interoperability of style containment implementations, and Chromium bugginess in its initial implementation
* Desire to improve the spec for counters (which was one of the main use cases motivating style containment)
* Desire to be able to ship layout and paint containment earlier in Firefox, without getting blocked on the above

Since then:
* There is a new, compelling use case in #6213.
* [This PR](https://github.com/whatwg/html/pull/4816) has landed to improve the spec
* Chrome has fixed the known problematic testcases
* Firefox did ship layout and paint containment, and an implementation is in progress in Webkit

Therefore I propose we remove the spec text mentioned above.

**Comment by astearns** [comment]
I think one thing we should consider is removing `contain: style` for lack of its own compelling use case and create a new `contain: query-target` (or some such) as suggested in #6174 that would include the behaviors currently defined for `style` `layout` and `size`

**Comment by chrishtr** [comment]
> for lack of its own compelling use case

I still think containing CSS counters is a compelling use case. Otherwise style recalc won't be contained.

**Comment by frivoal** [comment]
closed by f0da3020df9ea74126f79bd1b72120d7590719f6
